src/main/java/com/volunteer/enterprise/VolunteerAppController.java

```diff
@@ -94,46 +80,84 @@ public String saveVolunteer(Volunteer volunteer, Model model){
 
     /**
      * PostMapping for /volunteer endpoint
-     *Create a nwe volunteer with details provided via JSON
+     * Create a new volunteer with details provided via JSON
      *
      * Returns one of the following status codes:
-     * 201:
-     * 409:
+     * 201: Created
+     * 409: Conflict - Volunteer likely already exists
      * @param volunteer a JSON representation of Volunteer object to create
      * @return a newly created volunteer
      */
-
     @PostMapping(value="/volunteer", consumes = "application/json", produces = "application/json")
     @ResponseBody
     public Volunteer createVolunteer(@RequestBody Volunteer volunteer){
         Volunteer newVolunteer = null;
         try {
             newVolunteer = volunteerService.save(volunteer);
-        }catch (Exception e){
+        } catch (Exception e){
             e.printStackTrace();
         }
         return  newVolunteer;
     }
 
     /**
-     * Add comments here
-     *
+     * DeleteMapping for /volunteer/{id} endpoint
+     * Delete volunteer with provided ID
      *
-     *
-     *
-     * @param id
-     * @return
+     * @param id a unique identifier for volunteer to delete
+     * @return one of the following status codes:
+     *  200: Volunteer deletion success, even if volunteer didn't exist
+     *  409: Volunteer deletion error, likely provided malformed id
```

Comment:
Added detail to comments where doable to reduce lack of comments, a SonarQube Deadly Sin. [Reference](https://docs.sonarqube.org/display/SONARQUBE45/Not+Enough+or+Too+Many+Comments)

Reply:
Very good.